logging/build.go

```diff
@@ -0,0 +1,42 @@
+package logging
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"log"
+	"time"
+)
+
+var parentLogger *zap.Logger
```

Comment:
I think there might be a data race if `GetLogger` is called in parallel, e.g. if we decide to run tests in parallel in the future. Consider using `atomic.Pointer`

Reply:
delete this, we should use zap.L() only